docs/version0.46.md

```diff
@@ -77,6 +78,12 @@ When the `jcmd` tool sends a command to a running VM, the command might throw th
 
 The [`-Xtgc:allocation`](xtgc.md#allocation) option prints thread-specific allocation cache (TLH) statistics in addition to the cumulative allocation statistics.
 
+### New `-XX:[+|-]ShareOrphans` option added
+
+When `-Xshareclasses` was specified, only those class loaders that implemented the OpenJ9's public shared classes cache APIs (and its child class loaders) could store classes to the shared classes cache. Classes from custom class loaders that did not implement shared classes cache APIs were not stored to the shared classes cache because the VM did not have their module or class path information. You can enable class sharing from all class loaders, irrespective of whether the class loader implements the OpenJ9's public shared classes cache API, with the `-XX:+ShareOrphans` option.
```

Comment:
> Classes from custom class loaders that did not implement shared classes cache APIs were not stored to the shared classes cache because the VM did not have their module or class path information. 

Better to say: `Classes from custom class loaders that did not implement shared classes cache APIs were not stored to the shared classes cache. The JVM did not have their module or class path information. `